### ISSUE TITLE
Tab Center hotkeys completely remapped

### ISSUE THREAD

**Issue by ncalexan** [bwinton/TabCenter#815]
I have four pinned tabs.  All of a sudden, Alt-1 goes to the fourth pinned tab, and Alt-4 goes to the first.  Alt 5 goes to the *last* tab in the list, and Alt 9 goes to the 5th, which makes sense since that's where new tabs start.

Seriously, what on earth is going on?  You can't keep breaking my hotkeys!  (I've filed multiple bugs about keymappings/behaviour changing without warning.)  I did have some wackiness with Ctrl/Alt and keyboard entry doing things; did I somehow flip a setting or set Right-to-Left or something?

**Comment by ncalexan** [comment]
So, something went bonkers with my Nightly, and upgrading helped that a lot.  (Something re-organized my pinned tabs, which makes no sense but might explain why my key shortcuts were mangled.)

However, it seems something happened with the Tab Center behaviour as well: I can either have:
* "sensible shortcuts" and "new tab at bottom" (i.e., Ctrl+Tab moves _forward_ meaning vertically down)
* "unintuitive shortcuts" and "new tab at top" (i.e., Ctrl+Tab moves _backward_ meaning vertically up)
What happened to the behaviour I had and **liked**, where I had "sensible shortcuts" and "new tab at top"?

Can we make two preferences?

**Comment by ericawright** [comment]
We changed the way reverse tabs were handled thanks to discussion in these issues: 
https://github.com/bwinton/TabCenter/issues/655
https://github.com/bwinton/TabCenter/issues/709

*bonus* the new way we treat reverse tabs thankfully fixed this issue:
https://github.com/bwinton/TabCenter/issues/716

Of course, everyone has different preferences when working with their tabs and hopefully in the future when the "testing" phase of tab center has finished we will be able to expand the prefs and allow it to be much more customizable.

**Comment by SoftVision-PaulOiegas** [comment]
I agree the "Reverse Tab Order" option should not mes up with the keyboard shortcuts. The funniest part is with Ctrl + Page Down which goes up and vice versa. 😆

**Comment by indradhanush** [comment]
@ericawright Ping!

**Comment by bwinton** [comment]
@indradhanush: I'm not sure why you're pinging Erica, but no-one is working on this feature, and it isn't on the roadmap for the foreseeable future.  To be honest, I'm not even sure we would accept a patch if someone did the work of implementing it…
